backend/apps/owasp/templates/search/issue.html

```diff
@@ -155,6 +155,21 @@ <h5>How to tackle it</h5>
                     </div>
                 </div>
             </div>
+            <!-- Add Pagination Component -->
+            <div class="pagination">
+                <!-- Left Button: Decrement page -->
+                <button class="pagination-button"
+                        v-model="page"
+                        @click="decrementPage"
+                        :disabled="page === 0">Previous</button>
+                <!-- Page and Total Pages Display -->
+                <span>${this.page} / ${this.totalPages}</span>
```

Comment:
A couple of comments about the look and behavior:
  - could you put the buttons together in the center (just left and right of the page counter)
  - the page counter should start from `1` instead of `0` (I know Algolia start pagination from 0 🤷‍♂️).
  - I'd expect the app to scroll up to the top upon Previous/Next button click instead of showing the bottom page area
  
![Screenshot 2024-11-05 at 5 26 34 PM](https://github.com/user-attachments/assets/fd55d6d3-b06e-4e8d-9942-f00c9df6d5be)